apps/judicial-system/web/src/utils/validate.ts

```diff
@@ -257,6 +257,7 @@ export const isHearingArrangementsStepValidIC = (
 
 export const isProcessingStepValidIndictments = (
   workingCase: Case,
+  allFilesDoneOrError: boolean,
```

Comment:
This method is also used in `formHelper` so you probably need to set a default value here or add the new argument to the call in `formHelper`.

Reply:
Thanks! Good eye. This doesn't need to be a part of the validation function and has been removed.